src/Microsoft.AspNetCore.Authentication.Cookies/CookieAuthenticationMiddleware.cs

```diff
@@ -26,6 +30,13 @@ public class CookieAuthenticationMiddleware : AuthenticationMiddleware<CookieAut
                 throw new ArgumentNullException(nameof(dataProtectionProvider));
             }
 
+            if (antiforgery == null)
```

Comment:
It's interesting that this is required even if SuppressAntiforgeryValidation=true.